lib/matplotlib/tests/test_cbook.py

```diff
@@ -591,3 +591,29 @@ def test_warn_external(recwarn):
     cbook._warn_external("oops")
     assert len(recwarn) == 1
     assert recwarn[0].filename == __file__
+
+
+def test_array_patch_perimeters():
+    def check(x, rstride, cstride):
```

Comment:
Maybe add a comment that this is basically running the non-optimized version and comparing against it.